eng/pipelines/libraries/build-job.yml

```diff
@@ -66,7 +66,7 @@ jobs:
         - ${{ if and(eq(parameters.runtimeFlavor, 'coreclr'), eq(parameters.testScope, ''), ne(parameters.osGroup, 'WebAssembly')) }}:
           - _subset: clr.corelib+libs
         - ${{ if eq(parameters.runTests, true) }}:
-          - _subset: libs+libs.tests
+          - _subset: clr.corelib+libs+libs.tests
```

Comment:
I don't think we need to build corelib for net472, right?

Reply:
yeah probably isnt required, but wanted to keep the special casing to the minimum since building corelib is fairly fast.